app/views/items/show.html.erb

```diff
@@ -1,5 +1,11 @@
 <div class="item">
-  <h2><%= @item.title %></h2>
-  <h3><%= number_to_currency @item.currency %></h3>
-  <p><%= @item.description %></p>
+  <h2>Title: <%= @item.title %></h2>
+  <h3>Price: <%= number_to_currency @item.currency %></h3>
```

Comment:
Yes for the view there is, I will add a test in the model spec for that specific method